storage_service/common/tests/test_command_import_aip.py

```diff
@@ -0,0 +1,90 @@
+import os
+import uuid
+
+import pytest
+from django.core.management import call_command
+from locations import models
+
+TEST_DIR = os.path.dirname(os.path.realpath(__file__))
+FIXTURES_DIR = os.path.join(TEST_DIR, "fixtures")
+AIP_PATH = os.path.join(FIXTURES_DIR, "import_aip_test.7z")
+
+
+@pytest.fixture
+def aip_storage_location(db, tmp_path):
+    space_directory = tmp_path / "sub"
+    space_directory.mkdir()
+    space = models.Space.objects.create(
+        uuid=str(uuid.uuid4()),
+        path=space_directory,
+        access_protocol=models.Space.LOCAL_FILESYSTEM,
+        staging_path=space_directory,
+    )
+    pipeline = models.Pipeline.objects.create(uuid=str(uuid.uuid4()))
+    aipstore = models.Location.objects.create(
+        uuid=str(uuid.uuid4()),
+        space=space,
+        purpose="AS",
+        relative_path="",
+    )
+    models.Location.objects.create(
+        space=space, purpose=models.Location.STORAGE_SERVICE_INTERNAL, relative_path=""
+    )
+    models.LocalFilesystem.objects.create(space=space)
+    models.LocationPipeline.objects.get_or_create(pipeline=pipeline, location=aipstore)
+    return aipstore
+
+
+@pytest.mark.django_db
+def test_import_aip_output(mocker, capsys, aip_storage_location):
```

Comment:
If the test below tests that the package has been compressed, is it worth to test here that it isn't? And if so, should the name of the test be updated accordingly?

Reply:
sure. Thanks